book/chrome.md

```diff
@@ -270,7 +270,7 @@ browser might resolve this paradox with multi-phase layout.
 There are many considerations and optimizations of this kind that are
 needed to make text layout super fast.
 
-That method will pilfer the code from the old `flush` method. First,
+That method will pilfer code from the old `flush` method. First,
```

Comment:
"The" seems to imply all of it, like this is a command for the reader to do before moving on.